web/styles/modal.css

```diff
@@ -152,6 +152,11 @@
     .modal__title .stream-privacy-type-icon {
         width: 14px;
     }
+
+    .notification_stream_archive_warning {
+        margin-bottom: 0;
+        margin-top: 10px;
```

Comment:
I think we don't need `margin-top` anymore, and I'm not sure, but removing `margin-bottom` and the UI still looks decent.